docs/etcd-tls-assets.md

```diff
@@ -20,19 +20,16 @@ See also the [user-facing documentation](https://docs.openshift.com/container-pl
 ## etcd CA summary
 
 All etcd CAs and their CA bundles are stored in the `openshift-etcd` namespace. This NS is considered the source of truth for all certificates.
-To share CA bundles and client certificates with consumers (e.g. apiserver or the cluster-etcd-operator) they are copied by the `ResourceSyncController` into different places:
+To share CA bundles with consumers (e.g. apiserver or the cluster-etcd-operator) they are copied by the `ResourceSyncController` into different places:
 
-| CA (secret)                                      | CA bundle (configmap)                                         |
-|--------------------------------------------------|---------------------------------------------------------------|
-| openshift-config/etcd-signer (deprecated)        | openshift-config/etcd-ca-bundle                               |
-|                                                  | openshift-etcd/etcd-ca-bundle                                 |
-| openshift-etcd/etcd-signer (NEW)                 | openshift-etcd/etcd-ca-bundle                                 |
-| openshift-config/etcd-metric-signer (deprecated) | openshift-config/etcd-metric-serving-ca                       |
-|                                                  | openshift-etcd/etcd-metrics-ca-bundle                         |
-| openshift-etcd/etcd-metric-signer (NEW)          | openshift-config/etcd-metric-serving-ca (to be removed later) |
-|                                                  | openshift-etcd/etcd-metrics-ca-bundle                         |
+* openshift-etcd/etcd-ca-bundle (etcd server, configmap, source of truth)
+  * openshift-etcd-operator/etcd-ca-bundle (for the operator to reach etcd)
+  * openshift-config/etcd-serving-ca (for apiserver and others to connect to etcd)
+  * openshift-config/etcd-ca-bundle (just for consistency’s sake, should replace etcd-serving-ca, but is very cumbersome)
```

Comment:
Not suggesting a change but `just for consistency’s sake` made me realize I forgot who is consuming `openshift-config/etcd-ca-bundle` since it's not the operator, apiserver or etcd as they all have their own copies of the CA.

Was this required for somewhere in bootstrap? Trying to recall if there's more here that we need to expand on.

Reply:
good question, let me try to remove it and see what's breaking :) If it's not needed we'll just kick it out here too.

Reply:
seems unused. I leave this as its own commit, in case we need to revert it later.

Reply:
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_cluster-etcd-operator/1292/pull-ci-openshift-cluster-etcd-operator-master-e2e-aws-ovn-serial/1811018907415220224
```
[sig-api-machinery] API data in etcd [It] should be stored at the correct location and version for all resources [Serial] [Suite:openshift/conformance/serial]
  github.com/openshift/origin/test/extended/etcd/etcd_test_runner.go:33

    [FAILED] Unexpected error:
        <*errors.StatusError | 0xc000cb0e60>: 
        configmaps "etcd-ca-bundle" not found
        {
            ErrStatus: 
                code: 404
                details:
                  kind: configmaps
                  name: etcd-ca-bundle
                message: configmaps "etcd-ca-bundle" not found
                metadata: {}
                reason: NotFound
                status: Failure,
        }
    occurred
    In [It] at: github.com/openshift/origin/test/extended/etcd/etcd_test_runner.go:96 @ 07/10/24 14:58:27.997
```

We might be hitting this test preventing us from removing the configmap I think. Need to look more closely at what that test is doing or where we can update the test data. But in the interest of this PR we can also leave this as is.

https://github.com/openshift/origin/blob/bd4b3b87e16002820359555330093c05ef00ba52/test/extended/etcd/etcd_storage_path.go#L218-L222

Reply:
then let's roll it back for now. I've just also looked into the 4.14 branch, where we have pulled it from openshift-config:
https://github.com/openshift/cluster-etcd-operator/blob/release-4.14/pkg/operator/resourcesynccontroller/resourcesynccontroller.go#L27-L32

which means the render command must've put it there in the first place. Then let's leave it for backward compatibility :)

Reply:
reverted the commit and adjusted the documentation accordingly